Updated and rewrote DDEV instructions (#11)

StoppingBuck opened the pull request:
The current KB entry on DDEV development is outdated (#11) so this was originally an attempt at merely bringing it up to date with the [corresponding blog post](https://www.mautic.org/blog/developer/local-mautic-development-with-ddev), but it ended up containing some larger rewrites and additions in the hopes of making everything a bit clearer. Hopefully people agree :grinning: 

The MailHog part was not added, as MailHog is no longer available. It probably needs to be removed from the blog post as well.

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -49,19 +49,27 @@ Next it will ask for the docroot of the project.
 Since the index.php of Mautic is located in the root folder of the project, we can just go with the default value. Simply press enter.
 
-After this it will ask for the project type. Enter php and hit enter again.
+After this it will ask for the project type. Php, which we want to use, is the default choice. Simply press enter.
```
```suggestion
After this it will ask for the project type. PHP, which we want to use, is the default choice. Simply press enter.
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -49,19 +49,27 @@ Next it will ask for the docroot of the project.
-> Note: if you have Apache2 or nginx installed, and they are currently using port 80, ensure that you shut them down or change their ports before starting the DDEV instance. If you do not follow this step, starting will fail with an error message telling you that port 80 is already in use.
+Firstly, we change the default DDEV webserver to be Apache2 by changing the value of `webserver_type` to be `apache-fpm`.
+
+Secondly, we add a required PHP module by finding the line beginning with `webimage_extra_packages`, uncommenting it and settings its value to `[php7.3-imap]`. If you need any additional modules you can set them here as well (**make sure to replace 7.3 with whatever version of PHP is set as `php_version`**).
```
```suggestion
Secondly, we add a required PHP module by finding the line beginning with `webimage_extra_packages`, uncommenting it and setting its value to `[php7.3-imap]`. If you need any additional modules you can set them here as well (**make sure to replace 7.3 with whatever version of PHP is set as `php_version`**).
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -80,20 +88,69 @@ To stop the containers, simply run
-If you wish to directly execute a command inside the container without going in with ssh first you can use 
+The second option is executing the commands from **outside** of the DDEV container. 
+
+If you wish to directly execute a command in the container without first using SSH to connect to yet, you can use 
```
```suggestion
If you wish to directly execute a command in the container without first using SSH to connect, you can use 
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -106,26 +163,11 @@ and
 
 `ddev restart`
 
-### Using additional PHP modules
```
Was there a reason for removing this section on additional PHP modules?  Not sure if the omission was intentional!

StoppingBuck replied:
It was intentional :) Since adding extra modules was already covered previously in the guide (when the `php73-imap` was added), I felt it was confusing that it was suddenly re-mentioned down here, so instead I added a remark up there mentioning how the user can add any additional PHP modules they want.

RCheesley reviewed (changes requested):
Thanks for taking the time to do this - I have added a few grammar tweaks and a query about a deleted section for you to review :)

StoppingBuck commented:
> Thanks for taking the time to do this - I have added a few grammar tweaks and a query about a deleted section for you to review :)

Thank you for the feedback - I've adapted the changes accordingly :)

StoppingBuck commented:
Hey @RCheesley , three things:

1) I can see that despite me having approved all your changes (except one, which I explained my reasoning for), you are still listed as `1 change requested`. Maybe I'm just bad at GitHub, but... should I click somewhere to indicate that I have responded to the changes you requested? It seems a bit weird that the changes are still requested even after I have committed them :confused: 

2) I added a new section on using `mkcert` to enable HTTPS with DDEV, and linked to it during the Mautic installation step. My reasoning for this is that the moment a user visits mautic.ddev.site for the first time, they are greeted by a warning recommending them to turn on HTTPS so it makes sense to mention the possibility in that section :slightly_smiling_face: 

3) I also removed the bit about using `composer install` during the git cloning since it is covered in the part about SSH'ing into the container and running `composer install` there (as mentioned in the comments on the original blog post, it is recommended to run `composer install` from within the container instead of from the local machine to avoid trouble).

dennisameling reviewed (approved):
LGTM 👍

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
 
 [ddev]: (https://github.com/drud/ddev)
```
None of the link macros seem to be working, as you need to use the correct syntax (see the styleguide [here](https://docs.mautic.org/en/home/contributing#absolute-links)):

```suggestion
[ddev]: <https://github.com/drud/ddev>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
 
 [ddev]: (https://github.com/drud/ddev)
 [install-docker]: (https://ddev.readthedocs.io/en/stable/users/docker_installation/)
```
```suggestion
[install-docker]: <https://ddev.readthedocs.io/en/stable/users/docker_installation/>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
 [install-docker]: (https://ddev.readthedocs.io/en/stable/users/docker_installation/)
-[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
\ No newline at end of file
+[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
```
```suggestion
[install-ddev]: <https://ddev.readthedocs.io/en/stable/#installation>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
-[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
\ No newline at end of file
+[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
+[tz-values]: (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
```
```suggestion
[tz-values]: <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
\ No newline at end of file
+[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
+[tz-values]: (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+[composer]: (https://getcomposer.org)
```
```suggestion
[composer]: <https://getcomposer.org>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
+[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
+[tz-values]: (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+[composer]: (https://getcomposer.org)
+[cli-commands]: (https://docs.mautic.org/en/troubleshooting/command-line-tools-cli)
```
```suggestion
[cli-commands]: <https://docs.mautic.org/en/troubleshooting/command-line-tools-cli>
```

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
+[tz-values]: (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+[composer]: (https://getcomposer.org)
+[cli-commands]: (https://docs.mautic.org/en/troubleshooting/command-line-tools-cli)
+[mkcert]: (https://github.com/FiloSottile/mkcert)
```
```suggestion
[mkcert]: <https://github.com/FiloSottile/mkcert>
```

StoppingBuck replied:
Hey @RCheesley I can't approve this one since it's marked as "Outdated" for some reason? :confused:

RCheesley replied:
That means that the code in the PR has changed since the comment was left :) Will take a look!

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -36,36 +36,44 @@ Before we can get started you will need to install Docker and Docker Compose. Yo
+
+Finally, we set the timezone to be whatever timezone we are in. Find the line that starts with `timezone`, uncomment it and set its value to the tzdata format *`"Region/City"`* corresponding to whatever timezone you are in. Wikipedia hosts a [list of valid tz values][tz-values].
+
+> Note: if you have Apache2 or nginx running locally on your machine, and they are currently using port 80, ensure that you shut them down or change their ports before starting the DDEV instance. If you do not follow this step, starting DDEV will fail with an error message telling you that port 80 is already in use.
```
Let's put this part in a pretty blue 'note' box 😄 

```suggestion
>>>>> Note: if you have Apache2 or nginx running locally on your machine, and they are currently using port 80, ensure that you shut them down or change their ports before starting the DDEV instance. If you do not follow this step, starting DDEV will fail with an error message telling you that port 80 is already in use.
```

RCheesley reviewed (changes requested):
Thanks for taking a look - it will say that one review is pending when the person who requested changes hasn't yet re-reviewed after the changes are made.

I noticed by previewing your file using this link: https://github.com/mautic/mautic-community-knowledgebase/blob/020f553b296bf6d1a88f8e9c9fc2ba0eb1213a3a/pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md that none of the links are working.  

The syntax being used in the macros seems to be incorrect (we use `<>` for macros and `()` for inline links - see https://docs.mautic.org/en/home/contributing#absolute-links) so I've suggested those changes. 

Should then be good to go!

RCheesley commented on pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md:
```diff
@@ -134,6 +185,10 @@ A DDEV instance comes with PHPMyAdmin by default. To find out the location of th
+[tz-values]: <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>
+[composer]: <https://getcomposer.org>
+[cli-commands]: <https://docs.mautic.org/en/troubleshooting/command-line-tools-cli>
+[mkcert]: (https://github.com/FiloSottile/mkcert)
```
```suggestion
[mkcert]: <https://github.com/FiloSottile/mkcert>
```

RCheesley reviewed (changes requested):
Just this one to fix I think!

StoppingBuck commented:
Alright, I think I made it work :) Thank you for all the help!

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
Update modular.md

Updating the Contributing page

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

fakela committed:
(Grav GitSync) Automatic Commit from favour

RCheesley reviewed (approved):
Good to go!

RCheesley commented:
Hmm @StoppingBuck seems there is a conflict to resolve, are you able to take a look?

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
Update bottom.md

Changed the DB Quote in the Footer

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

Commit:
Merge branch 'master' of https://github.com/mautic/mautic-community-knowledgebase

LeonOltmanns committed:
Update bottom.md

Fixed wording

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
Update _blog.scss

Sidebar Uppercase

LeonOltmanns committed:
Update custom.css

TabloOfContent Top margin -20px

LeonOltmanns committed:
Update custom.css

text-transform uppercase //

LeonOltmanns committed:
Update _blog.scss

sidebar uppercase

LeonOltmanns committed:
Update _blog.scss

LeonOltmanns committed:
Update custom.css

text-tranform: none;

LeonOltmanns committed:
Update _blog.scss

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

LeonOltmanns committed:
(Grav GitSync) Automatic Commit from leon_oltmanns

StoppingBuck committed:
fix merge issues

StoppingBuck commented:
@RCheesley I apologize for the delay. I keep bumbling around with the Git commands but I think I finally managed to fix the conflict :smile: Is everything fine now?

StoppingBuck commented:
Welp, I think I rebased in a bad way...

RCheesley commented:
😱  Don't worry, happens to the best of us!

Am a tiny bit swamped with the 3.1.1 Release Candidate but will take a look tomorrow!
